apps/dotcms-ui/src/app/api/services/dot-crud/dot-crud.service.ts

```diff
@@ -57,7 +57,7 @@ export class DotCrudService {
      * @returns {Observable<any>}
      * @memberof DotCrudService
      */
-    getDataById(baseUrl: string, id: string, pick = 'entity'): Observable<any> {
+    getDataById(baseUrl: string, id: string, pick = 'entity'): Observable<unknown> {
```

Comment:
We said that services needs the type, you can check the endpoint response and transform that JSON to a type using this http://json2ts.com/

There is a vscode extension too.

Reply:
I might be wrong but this method is receiving the endpoint as `parameter`, doesn't that mean we don't know the exact data type since the `url` is dynamic?

Reply:
Exactly, that's why we need to use generics, where who consume it can say what's coming.